bin/www

```diff
@@ -8,4 +8,4 @@ require('babel-polyfill');
 require('babel-register');
 
 // Import the rest of our application.
-module.exports = require('../app.js');
+module.exports = require('../src/app.js');
```

Comment:
the server will not use the compiled code in production ?

Reply:
It definitely should :+1: